src/main/resources/assets/js/app/wizard/page/LiveFormPanel.ts

```diff
@@ -439,7 +439,7 @@ export class LiveFormPanel
         return this.pageView;
     }
 
-    setModel(liveEditModel: LiveEditModel) {
+    setModel(liveEditModel: LiveEditModel, showPanel?: boolean) {
```

Comment:
Do we ever call ``setModel()`` with ``showPanel = true``?
I found 1 case where we pass ``false`` and 2 cases where nothing is passed (so it's ``undefined``). So why do we need this parameter at all?
The same goes for ``clearSelection()``, actually - we pass either ``false`` or undefined, never ``true``.

![image](https://user-images.githubusercontent.com/11612490/54692326-2188d700-4b25-11e9-9d71-b34fa615e436.png)

Reply:
`InspectEvent` accepts `showPanel` param and set it to `true`, if it is `undefined`. So, setting a model with `undefined` value will result in calling the `InspectEvent` with the default (`true`) value. But the last time (see the screenshot) we explicitly set the `showPanel` param to `false`.

Reply:
Ok, I'll rephrase - why don't we do the same for ``showPanel`` argument in ``setModel`` and ``clearSelection`` methods in terms of declaring it with default ``true`` value instead of optional? It's very confusing now that ``undefined`` value of the ``showPanel`` argument in fact means ``true``.

Reply:
While the `undefined` (`?`) parameters were never meant to be of specific value, I can now see, that sometimes it can be confusing for the boolean values, when people assume, that `undefined` parameter will be equals to `false` in checks further in the code. I caught myself one time thinking this way.
Okay, I'll suggest that I'll be fixing this in another pull #338 since there were a lot of refactoring, covering some classes from this PR.
As for the `undefined` boolean parameters, let us follow this rule:
* Don't assign the `true` to the `undefined` parameters further in the call stack. `undefined` should be treated like `false` (`!!undefined`) instead:
```diff
function foo(silet?: boolean) {
  bar(silent);
}
function bar(silent?: boolean) {
  baz(silent);
}
- function baz(silent: boolean = true) {
-   if (silent) {
-     // ...
-   }
- }
+ function baz(silent: boolean = false) {
+   if (!silent) {
+     // ...
+   }
+ }
```
* Assigning `true` to boolean parameter is possible, but only when calling the function without the arguments, or with defined arguments:
```diff
function foo(silent: boolean = true) {
  // ...
}

function bar(value?: boolean) {
- foo(value);

+ foo();

+ foo(false);

+ const defined = true;
+ foo(defined);
}
```